Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ActionsTreeView.cs

```diff
@@ -208,6 +210,81 @@ public override void RedrawUI(ViewState viewState)
             m_PropertiesScrollview.visible = m_ActionsTreeView.GetTreeCount() > 0;
         }
 
+        private void OnDraggedItem(DragPerformEvent evt)
+        {
+            bool discardDrag = false;
+            foreach (var index in m_ActionsTreeView.selectedIndices)
+            {
+                var treeViewItem = m_ActionsTreeView.panel.Pick(evt.mousePosition)?.GetFirstAncestorOfType<TreeView>();
```

Comment:
Surprised to see some kind of mouse picking is needed here (feels brittle?), isn't possible to rely on currentTarget and target for this? (I have no experience in UITK drag and drop but that's usually how this stuff works, with a drag source and a drag target). Basically I believe I do not really understand how this function operates, would it be possible to add some inline comment explaining this or feedback into this review PR?

Reply:
I added a inline commit. We need to pick the item from the panel as the event is registered to the target (the TreeView), so the target and the current target are always in this panel. We can't choose another element to receive the events, because we want to use the reorderable logic from UITK. Picking the elements was something that was recommended by the UI-Toolkit team, find the related thread [here](https://unity.slack.com/archives/C3414V4UV/p1692199097910419?thread_ts=1692181020.348629&cid=C3414V4UV).